packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2943,7 +2944,7 @@ export const audiusBackend = ({
       const userBank = await audiusLibs.solanaWeb3Manager.deriveUserBank()
       const ownerWAudioBalance =
         await audiusLibs.solanaWeb3Manager.getWAudioBalance(userBank)
-      if (!ownerWAudioBalance) {
+      if (isNullOrUndefined(ownerWAudioBalance)) {
         throw new Error('Failed to fetch account waudio balance')
```

Comment:
This would throw an error even if the WAudio balance was 0. Not an issue before #6185 because we used to equate null (error) to 0 in UI, but this is now problematic.